ml-agents/mlagents/trainers/tests/test_simple_rl.py

```diff
@@ -323,3 +330,43 @@ def test_simple_ghost_fails(use_discrete):
     assert any(reward > success_threshold for reward in processed_rewards) and any(
         reward < success_threshold for reward in processed_rewards
     )
+
+
+@pytest.mark.parametrize("use_discrete", [True, False])
+def test_simple_record(use_discrete):
```

Comment:
We don't actually need to run this test every time since the .demos are written to a file but it does test the new write demo functions etc.